R/render.r

```diff
@@ -482,7 +487,13 @@ check_bootswatch_theme <- function(bootswatch_theme, bs_version, pkg) {
 }
 
 pkgdown_bslib_defaults <- function() {
-  list(`navbar-nav-link-padding-x` = "1rem")
+  list(
+    `navbar-nav-link-padding-x` = "1rem",
+    `primary` = "#0054AD",
+    `secondary` = "#e9ecef",
```

Comment:
actually here I'd like to use `$gray-200` but I did not find how.

Reply:
Actually #e9ecef is a light grayish blue so it's not a mix of black and white.

If it were a mix, it'd need to be defined _after_ the defaults actually, with `bslib::bs_get_variable()` to get `$black` and `$white` as apparently I can't just write the variable names in there.